lang/java/reef-bridge-java/src/main/java/org/apache/reef/javabridge/JavaBridge.java

```diff
@@ -18,6 +18,9 @@
  */
 package org.apache.reef.javabridge;
 
+/**
+ * TODO[JIRA REEF-383] Document/Refactor JavaBridge.
```

Comment:
The one I use :-) Parent JIRA for this issue https://issues.apache.org/jira/browse/REEF-519 has it